adobe/aio-cli-plugin-app-dev
# feat: ACNA-2783 - front-end hot reload

This enables front-end hot reload.
This PR incorporates changes from original PR #14 (which has been closed)

## How Has This Been Tested?

- local link of the plugin
- debug in VS Code of an AB project after running `aio app dev init`
  1. choose the "Debug" tab in the sidebar
  2. choose the "App Builder: debug full stack" configuration in the Debug dropdown menu
  3. select the green "Play" button
  4. wait for the Chrome browser (required) to launch
  5. edit any Javascript or HTML in the web-src folder
  6. observe the changes reflected in the browser
- unit tests are expected to fail since we haven't completed unit tests yet

## Types of changes




## Checklist:
